### DISCUSSION TITLE
Our understanding of the audience needs for the TTW book

### DISCUSSION THREAD

**Discussion by aranas** [the-turing-way/the-turing-way#3713] [Q&A]
This discussion is related to issue #3680  of SEO optimization for the TTW book.

Although the [FAIR](https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-fair.html?highlight=fair) principles have been developed in relation to data management, Findability is crucial for open science more generally as research that is not findable, is automatically also not accessible. In order to ensure that the TTW content reaches the open science community, we need to ensure the following:
 1) That our content is tailored to the needs and preferences of the community. 
 2) That our content is discoverable for the community, especially newcomers

Because the TTW book is created through community collaboration, we can assume 1) is likely met. Although it might be that content we create for a subgroup within the community does not speak to other subgroups.

Some Questions:

- **How well do you feel you currently understand the TTW audience’s needs and preferences?** 

- **As an author what would you like to know about how readers interact with your content?**

Now, making sure that our content is tailored does not necessarily mean it will reach all of the relevant members of the community. It might be that due to third-party curation, our TTW content is not prioritized. Here we can rely on data to better understand what is being shown where and adjust our content (through keywords & metadata) to be connected to the right readers.

- **When writing a chapter, do you think about discoverability?**
- **Have you taken any steps to actively improve discoverability of your content?** (eg through using specific keywords etc)?

**Comment by EstherPlomp** [comment]
I think these are nice questions to ask, and they don't really relate to the FAIR principles as these principles are more about technical accessibility and not really about the content itself. So if you're writing things up I would leave that part of the issue out! 

- I think I have a good understanding of the needs/preferences of the audiences; in general these are people interested in data science that want to learn more about a specific topic. 
- It would be great to have a bit more information about how people interact with the content. I have a general feeling for this regarding the data management parts because I use these in workshops or I see parts of them back in conference presentations or in other people's workshop slides and so forth. 

I'm not that interested in using keywords that will result in better search engine results. Search is pretty broken right now and I don't think we should cater to this and instead rely on community promotion of the materials. Plus if you look for the keywords used in chapter titles the Turing Way often shows up anyway, especially if you use The Turing Way in the search. In short: I think we got this covered!

**Reply by aranas** [reply]
Interesting that FAIR may be considered to be restricted to technical accessibility. I was wondering if the leap to FAIR was maybe a stretch. If I were to write this down I was thinking this could fit under the chapter "Communicating Accessibly", let me know if you have another suggestion of where something like this could fit!

**Comment by AbcSxyZ** [comment]
First time contributor to TTW and working on a new chapter for open source (https://github.com/the-turing-way/the-turing-way/pull/3705), to share some thoughts on the current approach. One important element is that there are debates on the concept, it impacts some considerations.

> When writing a chapter, do you think about discoverability ?

Not from an SEO perspective, but I think more about how the result may be shared and discussed, by bringing questioning to people. More thinking about the social interactions that can lead to discover this content rather than a technical approach.

> How well do you feel you currently understand the TTW audience’s needs and preferences?

Quick aside, I'm first wondering how the TTW team is aware of its own audience. Beyond the fact that it is a research community with important relation to openness, needs and preference remains vague for me. I believe there is interest because open source is a key notion in these dynamics and it's hard to explain, it's an assumption that it would be useful from a personal knowledge of the field (+ I interact with some members of the community which shows a certain relevance/interest).

What may be also particular is that this page on open source would not only in direction of TTW "community" but to other open source specialist (not even researchers) as there is this ongoing debates on the concept. Kind of attempt to reach a specific community on this specific element.

> As an author what would you like to know about how readers interact with your content?

I think in this case what may meaningful is PR/issues/improvements/discussions emerging around the content as there is this intent to build the understanding of the concept. We were discussing about how to make the TTW a place for actual debates, the interest will be in seeing the interactions that this generates.

I suppose SEO is more asked from the book perspective, but issues/PR and these other elements can be important to consider and will be also discovered through search engines. Do we consider SEO when writing issues/PR ? I doubt this is the case 😅

P.S.: On the FAIR question with @EstherPlomp, I would think this is about findability of the resource.

**Reply by aranas** [reply]
@AbcSxyZ very interesting points and yes indeed the SEO discussion here was meant to be mainly about the TTW book. I have now adjusted the title to better reflect that. But actually I find your point about considering SEO when writing issues equally interesting. Good PR titles & tags certainly will influence how new TTW members engage and with so many issues it can quickly become overwhelming

**Reply by AbcSxyZ** [reply]
These meta elements could sometimes be seen a lot externally, with a public interest depending on the topic when questioning are shared/meaningful. In software when mailing list are used it can be a specific email, and so on.

Good titles will definitively help inside github search engine, through more traditional one like google, would be great to have it more in mind. I see it more as a need for clarity in ideas. It could be important for people having similar problems when it's discussed in existing issues. Some inspiration can be found with [stackoverflow](https://stackoverflow.com/) (Q&A for IT questions, indexed on search engines, you should ask a dev how they use it if you don't already know it).

**Comment by dingaaling** [comment]
Thanks for starting this conversation @aranas! Some comments from my experience:

- Re: discoverability, I have tried to write for a general audience, as I believe is the general tone/aim of The Turing Way. However, in practice I do find that for my specific chapters (esp. ML Model Licensing) there are every specific kinds of people that are finding, commenting, and engaging with the topics. These are primarily open source software or open source research organisations, who have rightfully pointed out corrections and directly made edits. This has made me wonder how to orient certain "actively developing" chapters towards more ongoing community conversation/collaboration
- As an author, I'd be curious what parts of the chapter people are spending the most time on and what kind of activities they are taking (e.g. linking/sharing/citing/critiquing)

**Comment by Arielle-Bennett** [comment]
**How well do you feel you currently understand the TTW audience’s needs and preferences?**
- I think I have a good general idea, most people are looking for accessible (in terms of technical language/coding use) material on a variety of topics around research practices. They come from a wide variety of different fields and tend to be earlier in their careers
- I'd be interested to explore more about specifics of whether their expectations and needs match what they find in the book!

**As an author what would you like to know about how readers interact with your content?**
- I think how they get to the page, how long they spend, and where they go when they leave are useful, they're okay proxies at scale for whether the content is interesting and relevant, and whether it sparks inspiration to explore the book more or do further reading
- However, it's not to possible to do with analytics, but I'd be most interested to hear from folks who have used the content in the book as inspiration for their own work and how they've engaged with it as part of focus groups

**When writing a chapter, do you think about discoverability?**
- Sometimes! But I'm mostly writing for understanding when someone lands on the page, not the discoverability itself if that makes sense

**Have you taken any steps to actively improve discoverability of your content? (eg through using specific keywords etc)?**
- I try to make sure I have good quality links out for most pages and use clear, less technical language which helps with SEO, but I don't think about specific keywords beyond using the appropriate terms. Also, the good quality links and language choices just help with readability and content quality anyway!
- However, I could see adding extra phrases being helpful for specific landing pages. I'm thinking specifically of Research Infrastructure Roles chapter - this group of roles are referred to by a lot of different names and phrases even within the UK, so specifically adding those terms into the landing page might be helpful to surface it in more searches.

**Comment by JimMadge** [comment]
> How well do you feel you currently understand the TTW audience’s needs and preferences?

My feeling is the audience is very broad, so clarity and accessibility are a priority when writing anything.
It also means it's OK to cover a niche topic that won't be interesting to all readers.

> As an author what would you like to know about how readers interact with your content?

The common metrics like page views, time spent, where people came from would be useful (as long as we actually look at the numbers and think about them :smile:).
I think it would also be interesting to have an idea of language preference. Are we able to find out what language a user would prefer (and not just which language they are reading)?

> When writing a chapter, do you think about discoverability?
> Have you taken any steps to actively improve discoverability of your content? (eg through using specific keywords etc)?

Not beyond placing a chapter in a sensible place and adding links to/from related chapters.

I would be good to know how important SEO would be for reaching new and existing readers.
Do people look for the kind of information we provide using search engines?
Would it help current readers find sections of the book they didn't know about?

I'm not sure if it is really possible to answer that question but I'm wondering if we had to choose between SEO and promotion through networks or word of mouth, which would be more effective.
